Enable sklearnex documentation (#587)
intel/scikit-learn-intelex: doc/sources/gpu.rst

.. _gpu:

###########################################################################

> oneAPI and GPU support in Intel(R) Extension for Scikit-learn* with daal4py

Reviewer: ```suggestion
oneAPI and GPU support in Intel(R) Extension for Scikit-learn* with Intel(R) Extension for Scikit-learn*
```

###########################################################################

> daal4py is installed with Intel(R) Extension for Scikit-learn* and provides support of oneAPI concepts such as context and queues, which means that

Reviewer 2: ```suggestion
daal4py is installed with Intel(R) Extension for Scikit-learn* and provides support of oneAPI concepts, such as context and queues, which means that
```

> algorithms can be executed on different devices, GPUs in particular. This is implemented via 'with sycl_context("xpu")'

Reviewer 3: ```suggestion
algorithms can be executed on different devices, GPUs in particular. This is implemented via ``with sycl_context("xpu")``
```
